Update membership FAQ v2 (#3)
ErinBecker/carpentries.org: pages/member_faq.md

self-organised workshops and access to Member pricing for additional Centrally-Organised workshops and Instructor Training seats purchased during the membership year. Read more about the benefits and dues for each plan on our [membership](/membership) page. 

2. **Which membership should I purchase?**<br>

> * Bronze membership is an excellent option for any organisation starting out with The Carpentries, who want to quickly bring data and computational training to their community. The Bronze level is also great for introducing your local community to The Carpentries and building interest for instructor training.

Reviewer: ```suggestion
* Bronze membership is an excellent option for any organisation starting out with The Carpentries and wanting to quickly bring data and computational training to their community. The Bronze level is also great for introducing your local community to The Carpentries and building interest for instructor training.
```

* Silver and Gold memberships are ideal for organisations wanting to quickly build capacity for training in their community. 

> * Platinum memberships are best for organisations wo uld like to continue building capacity for training and already have a group of local Carpentries Instructors capable of running Self-Organised workshops. 

Reviewer: ```suggestion
* Platinum memberships are best for organisations that would like to continue building capacity for training and already have a group of local Carpentries Instructors capable of running Self-Organised workshops. 
```

Right away! You can request workshops as soon as your membership agreement is in place by completing [this form]({{ site.wrf_landing }}). To allow us time to recruit Instructors and give them time to plan the workshop with you, we generally require 2-3 months lead time to schedule a workshop.  

3. **Can I run my own workshops without organisational support from The Carpentries?**<br>

> Yes! Once you start building a team of certified Instructors, you can work with them to run Self-Organised workshops. We ask that Instructors let us know when they are planning a Slef-Organised workshop in advance by completing this [self-organised notification form](https://amy.carpentries.org/forms/self-organised/). This form initiates a process so that The Carpentries staff can support the workshop by providing survey links and tracking of Instructor and Helper participation, as well as publicising the workshop on Carpentries websites, if desired. As long at least one Instructor is Carpentries-certified, and either the Data Carpentry, Library Carpentry, or Software Carpentry core content is taught, the workshop can be labeled a Carpentries workshop. There is no limit to the number of Self-Organised workshops you can run, regardless of membership level. Self-Organised workshops do not count against the number of workshops included in your membership agreement. 

Reviewer: ```suggestion
Yes! Once you start building a team of certified Instructors, you can work with them to run Self-Organised workshops. We ask that Instructors let us know when they are planning a Self-Organised workshop in advance by completing this [form](https://amy.carpentries.org/forms/self-organised/). This form initiates a process so that The Carpentries staff can support the workshop by providing survey links, tracking Instructor and Helper participation, and publicising the workshop on The Carpentries websites, if desired. As long as at least one Instructor is Carpentries-certified, and either the Data Carpentry, Library Carpentry, or Software Carpentry core content is taught, the workshop can be labeled a Carpentries workshop. Self-Organised workshops do not count against the number of workshops included in your membership agreement and there is no limit to the number of Self-Organised workshops you can run. 
```

non-member affiliated trainees. 

3. **How soon after signing my membership can I get started with Instructor Training?**<br>

> In your 12 month membership period, we will be offering at least 6-8 online events scheduled across various timezones. To view a full list of upcoming training events please see the [Instructor Training Calendar page](https://carpentries.github.io/instructor-training/training_calendar/). Your trainees are invited to join any event(s) that suits their schedule(s), either as a group or individually, along with new trainees from other parts of our community.<br>

Reviewer: ```suggestion
In your 12 month membership period, we will be offering at least 6-8 online events scheduled across various timezones. To view a full list of upcoming training events please see our [Instructor Training Calendar](https://carpentries.github.io/instructor-training/training_calendar/). Your trainees are invited to join any event(s) that suits their schedule(s), either as a group or individually, along with new trainees from other parts of our community.<br>
```

> In your 12 month membership period, we will be offering at least 6-8 online events scheduled across various timezones. To view a full list of upcoming training events please see the [Instructor Training Calendar page](https://carpentries.github.io/instructor-training/training_calendar/). Your trainees are invited to join any event(s) that suits their schedule(s), either as a group or individually, along with new trainees from other parts of our community.<br>

Reviewer: ```suggestion
In your 12 month membership period, we will be offering at least 6-8 online events scheduled across various time zones. To view a full list of upcoming training events please see the [Instructor Training Calendar page](https://carpentries.github.io/instructor-training/training_calendar/). Your trainees are invited to join any event(s) that suits their schedule(s), either as a group or individually, along with new trainees from other parts of our community.<br>
```

> It is difficult to schedule events in everytime zone, so your trainees are always welcome to join scheduled online events in adjacent timezones, given the times are reasonable for their schedule. However, knowing this may not be ideal for you, we will work with you to schedule an online event based on your needs and our Trainers’ availability. To ensure that we can meet everyone’s needs, we generally like to schedule these events 2-3 months in advance. Once this event is scheduled, we may invite people from other sites to join as well, to begin to build community in your part of the world.<br>

Reviewer: ```suggestion
It is difficult to schedule events in every time zone, so your trainees are always welcome to join scheduled online events in adjacent time zones, given the times are reasonable for their schedule. However, knowing this may not be ideal for you, we will work with you to schedule an online event based on your needs and our Trainers’ availability. To ensure that we can meet everyone’s needs, we generally like to schedule these events 2-3 months in advance. Once this event is scheduled, we may invite people from other sites to join as well, to begin to build community in your part of the world.<br>
```
